backend/app/controllers/api/v1/project_developers_controller.rb

```diff
@@ -23,6 +26,17 @@ def show
         ).serializable_hash
       end
 
+      def create
+        current_user.with_lock do
+          raise API::UnprocessableEntityError, I18n.t("errors.messages.user_multiple_accounts") if current_user.account_id.present?
```

Comment:
I wasn't really sure what should happen when user already have account -- I decided to raise error :)

Reply:
Great choice :+1: